package.json

```diff
@@ -68,21 +68,24 @@
     "npm": ">= 5.3.0"
   },
   "dependencies": {
-    "@angular/core": "^4.4.0",
-    "@angular/http": "^4.4.0",
-    "lodash": "^4.17.4",
+    "lodash": "^4.17.4"
+  },
+  "peerDependencies": {
```

Comment:
so how does it work when adding `ngx-base` to the Peer deps?

Reply:
It means that the consumer must install ngx-base. I think it makes sense to only want a single base in the system, no?